inc/JACDAC/JDProtocol.h

```diff
@@ -674,6 +729,22 @@ namespace codal
          **/
         int setBridge(JDDriver* bridge);
 
+        /**
+         * Set the name to use for error codes and panics
+         *
+         * @param s the name to use for error codes and panic's
+         *
+         * @note Must be 6 characters or smaller.
+         **/
+        static int setDebugName(ManagedString s);
```

Comment:
is this the "device" debug name? (e.g. same for all drivers on the same device)

Reply:
Yes